packages/common/core-interfaces/src/handles.ts

```diff
@@ -101,6 +101,22 @@ export interface IFluidHandleInternal<
 	bind(handle: IFluidHandleInternal): void;
 }
 
+/**
+ * @privateRemarks
+ * To be merged onto IFluidHandleInternal in accordance with breaking change policy
+ * @internal
+ */
+export interface IFluidHandleInternalPlaceholder<
```

Comment:
I fear the term "Placeholder" will cause confusion because a placeholder is something that is eventually replaced, this handle will not be replaced if/when the blob for it is uploaded. That's in fact its core value-add: instead of representing the lack of handle with a sentinel value that then gets replaced, this handle remains in place.

Reply:
I think other names could also be fine - did you have a preference in mind?

Reply:
Nothing I particularly love, unfortunately: "BlobSeedHandle", "BlobPromiseHandle". "BlobHandle" was my original suggestion, but I see that as giving up on trying to find name that hints at the contract. So, I think "BlobPromiseHandle" is the option I find least objectionable for now. Open to other ideas.

Reply:
Chatted with @yann-achard-MS  - since "placeholder" implies that we have a thing that can substitute "in place" of the real payload, it seems misleading because we actually have nothing until the blob upload completes.  And the handle itself is not a placeholder to be swapped out either, as he mentioned above.  Went with payloadPending in latest as this is more accurate/descriptive of what it signifies and what is specifically pending, plus aligns fairly well with `payloadState` in the observability API surface.